webpack.config.js

```diff
@@ -111,6 +111,7 @@ if (isProd) {
     watchFiles: ['src/*'],
     headers: {
       'Access-Control-Allow-Origin': '*',
+      "Access-Control-Allow-Headers": "baggage, sentry-trace"
```

Comment:
For what reason do we need? 👀

Reply:
It was needed for dev server to work when using with remnote opened in a browser. See in the remnote template: https://github.com/remnoteio/remnote-plugin-template-react/commit/a9c1590787e6a4ea0bb5fb4610da4547edea9866

Reply:
I got it! I suppose remnote uses Sentry to track bugs and those headers are required to do it.